internal/pkg/deploy/cloudformation/stack/static_site.go

```diff
@@ -101,6 +101,13 @@ func (s *StaticSite) Template() (string, error) {
 			return "", err
 		}
 	}
+	// Note: Unlike NLB, App Runner, and LBWS, uses AWS::Route53::RecordSetGroup to
+	// create the A-Record to route traffic to the Load balancer endpoint.
+	// Where as Static Site default domain alias is created using custom resources.
+	staticSiteAlias := fmt.Sprintf("%s.%s.%s.%s", s.name, s.env, s.app, s.appInfo.Domain)
```

Comment:
just to clarify - what happens if the app doesn't have a domain? don't we not want to set `staticSiteAlias` in that case?

Reply:
That's a good point. I will change the logic to catch this case. Thank you for the catch.

Reply:
Address this in this [commit](https://github.com/aws/copilot-cli/pull/5191/commits/6dd50ee93582b6adfd0fc3997501c313f1e0b1d4)

Reply:
1. If app has no `domain` and no `http.alias` then the URL will be `http://d1vk2xptkdkbf8.cloudfront.net`
2. If app has domain associated and no `http.alias` then the URL will be `https://d1vk2xptkdkbf8.cloudfront.net` and `https://svc.app.env.domain`
3. If app has domain and http.alias=example.com then the URL will be `https://d1vk2xptkdkbf8.cloudfront.net` and `https://example.com`